job/validators.py

```diff
@@ -209,42 +209,42 @@ def cross_clean(self):
         PV tilt set to latitude if not provided and prod_factor_series validated
         """
         def cross_clean_pv(pvmodel):
-            if pvmodel.__getattribute__("tilt") == 0.537:  # 0.537 is a dummy number, default tilt
-                pvmodel.__setattr__("tilt", self.models["Site"].__getattribute__("latitude"))
+            if pvmodel.__getattribute__("tilt") == 0.537:  # 0.537 is a dummy number, default tilt # TODO: change to None? 
```

Comment:
@rathod-b The PV tilt defaults should use the same logic as the Julia pkg (based on array type).

From the Julia pkg: `tilt::Real= array_type == 1 ? 10 : abs(latitude), # tilt = 10 deg for rooftop systems, abs(lat) for ground-mount` 

Could you update this?

Reply:
Sure, I can update this.